### DISCUSSION TITLE
Using enums does not work with query()->firstWhere

### DISCUSSION THREAD

**Discussion by ismailnakkar** [laravel/framework#39838] [General]
Laravel Version: 8.74.0
PHP Version: 8.1.0
Database Driver & Version: -

Description:

Using enums does not work with query()->firstWhere in DB query builder

Steps To Reproduce:
Use firstWhere with the new enums of php8.1

**Comment by henzeb** [comment]
It still does not work in Laravel 9.17, but I have made functionality that proxies the enum so that it is stringable. see https://github.com/henzeb/enumhancer/blob/main/docs/functions.md#backing
